lib/error/error_handler.rb

```diff
@@ -0,0 +1,98 @@
+    def self.included(clazz)
+      clazz.class_eval do
+        rescue_from ActionController::RoutingError do |error|
+          respond(:routing_error, 404, error.to_s)
+        end
+
+        rescue_from ActionController::MethodNotAllowed do |error|
+          respond(:method_not_allowed, 405, error.to_s)
+        end
+
+        rescue_from ActionController::UnknownHttpMethod do |error|
+          respond(:unknown_http_method, 405, error.to_s)
+        end
+
+        rescue_from ActionController::NotImplemented do |error|
+          respond(:not_implemented, 501, error.to_s)
+        end
+
+        rescue_from ActionController::UnknownFormat do |error|
+          respond(:unknown_format, 406, error.to_s)
+        end
+
+        rescue_from ActionController::BadRequest do |error|
+          respond(:bad_request, 400, error.to_s)
+        end
+
+        rescue_from ActionController::ParameterMissing do |error|
+          respond(:parameter_missing, 400, error.to_s)
+        end
+
+        rescue_from ActiveRecord::RecordNotFound do |error|
+          respond(:record_not_found, 404, error.to_s)
+        end
+
+        rescue_from ActiveRecord::RecordInvalid do |error|
+          respond(:record_invalid, 422, error.to_s)
+        end
+
+        rescue_from ActiveRecord::RecordNotSaved do |error|
+          respond(:record_not_saved, 422, error.to_s)
+        end
+
+        rescue_from ActiveRecord::RecordNotDestroyed do |error|
+          respond(:record_not_destroyed, 422, error.to_s)
+        end
+
+        rescue_from ActiveRecord::RecordNotUnique do |error|
+          respond(:record_not_unique, 422, error.to_s)
+        end
+
+        rescue_from ActiveRecord::InvalidForeignKey do |error|
+          respond(:invalid_foreign_key, 422, error.to_s)
+        end
+
+        rescue_from ActiveRecord::MismatchedForeignKey do |error|
+          respond(:mismatched_foreign_key, 422, error.to_s)
+        end
+
+        rescue_from ActiveModel::UnknownAttributeError do |error|
+          respond(:unknown_attribute_error, 422, error.to_s)
+        end
+
+        rescue_from ActiveModel::StrictValidationFailed do |error|
+          respond(:strict_validation_failed, 422, error.to_s)
+        end
+
+        rescue_from ActiveModel::RangeError do |error|
+          respond(:range_error, 422, error.to_s)
+        end
+
+        rescue_from AASM::UndefinedState do |error|
+          respond(:undefined_state, 422, error.to_s)
+        end
+
+        rescue_from AASM::InvalidTransition do |error|
+          respond(:invalid_transition, 422, error.to_s)
+        end
+
+        rescue_from ArgumentError do |error|
+          respond(:argument_error, 404, error.to_s)
+        end
+
+        rescue_from StandardError do |e|
+          respond(:standard_error, 500, e.to_s)
+        end
+      end
+    end
+
+    private
+
+    def respond(error, status, message)
+      json = Helpers::Render.json(error, status, message)
+      render json: json, status: status
```

Comment:
GJ! but i think u need a empty line under 94, what do you think?
![image](https://user-images.githubusercontent.com/70888057/156213768-d26b37b4-0784-4f35-ba73-5a17bf339361.png)

Reply:
it's not an endpoint, it's a function